src/ansys/fluent/core/solver/flobject.py

```diff
@@ -971,7 +971,7 @@ def get_root(flproxy, version: str = "") -> Group:
             )
             raise RuntimeError("Mismatch in hash values")
         cls = settings.root
-    except (ImportError, RuntimeError):
+    except Exception:
```

Comment:
The empty settings directory may remain in site-packages after uninstallation, then it throws AttributeError from `settings.SHASH` from within the try block. Catching the general Excepton here as we need to catch too many different types.

Reply:
Yes, but then you break the `raise RuntimeError(...)` three lines above no?

Reply:
I think the function should still proceed to `get_cls` when the hash-value is mismatched as then the settings classes can be built in-memory from the metadata coming from the running server session. I'll log a warning in this case so that the message is still printed.